github/ipfs.yml

```
@@ -3621,21 +3637,18 @@ repositories:
         - JS Core Team
         - Repos - JavaScript
     visibility: public
-  js.ipfs.io:
+  js.ipfs.tech:
     branch_protection:
       master: {}
```

Bringing back branch protections which weren't created.
```suggestion
      master: {}
      release: {}
      release-*: {}
```